LeanbackShowcase/README.md

```diff
@@ -32,25 +32,24 @@ If you use Android Studio as recommended, the following dependencies will **auto
 - Clone this repo:
 
 ```sh
-git clone https://github.com/googlesamples/leanback-showcase.git
+git clone https://github.com/android/tv-samples.git
 ```
 
 - In the local.properties file add the location to your Android SDK (You need to install Android SDK 'N').
 - Open the project in [Android Studio][studio].
-- Compile and deploy to your Android TV emulator or device (such as a Nexus Player).
+- Compile and deploy to your Android TV device (such as an ADT-2 or ADT-3).
```

Comment:
A developer should still be able to use an emulator. Often they forget they have that ability so let's not remove a chance where we can ingrain that into them :)

Reply:
Good point, I'll update it across all the samples

Reply:
I don't think I was clear. Let's mention the emulator and **not** remove it from the message.

Reply:
You were :) I think it's just showing you a single commit of the PR. I updated it to include the emulator across the three individual project READMEs in the third commit to this PR. Github anchors comments to their line in the code at that point in time, so commits added to the PR that change the line after that don't show up when viewing the comment (but do if viewing the individual commit or the overall files changed tab). See https://github.com/android/tv-samples/pull/16/files for clarity.

Reply:
Ah gotcha, I am so used to amended commits that I forgot how Github's PR work. Thanks for your patience!